### ISSUE TITLE
Question about recurrence

### ISSUE THREAD

**Issue by p2635** [mpcjanssen/simpletask-android#892]
Hi all, I have a task called 'pack lunch' recurring everyday. When I tick it off as complete, it shows another one but I only want to see this _tomorrow_.

I'm a newbie so I'm still learning how to use this. What is the easiest way to get it to show only once everyday?

**Comment by smichel17** [comment]
There's two ways to do it.

1. The "normal way" with threshold dates.
    a. Add a new task, "pack lunch rec:1d".
    b. Set the threshold date for today.
    c. [optional] Set the due date for today.
    d. Make sure "Show tasks with threshold date in future" is not checked in the filter.
   e. When you complete this task, it should create a new, identical task with the threshold and due dates shifted to tomorrow.
2. The advanced way -- use only a due date, and use a lua script to show only tasks due today.
    - You probably don't need this so I'm not going to give instructions right now.
    - The advantage is that you can use threshold dates for something else or use more complicated logic. For example, my main filter shows tasks where more than half the time between their threshold and due dates has elapsed.

**Comment by p2635** [comment]
@smichel17  thanks, it works!

sorry I have another one... Using the same example, if i'm packing lunches only for this week Monday - Friday how do I make it stop recurring after this Fritday? (i'm guessing make it due this Friday?)

**Comment by smichel17** [comment]
There's no built in way to stop recurring at a certain date, since recurrence adjusts all dates associated with the task. The easiest way is probably just to delete the one that's created when you mark Friday's task complete.

**Comment by p2635** [comment]
@smichel17 you've been amazingly helpful, thank you, this can be closed. I love using this simple text file system

**Comment by mpcjanssen** [comment]
As a closing remark. I could add an onComplete Lua callback which will return a new tasks so you can make the recurrence completely custom. See #893.
